bwmon/www/bwmon.html

```diff
@@ -146,11 +146,16 @@
 							<label class="btn btn-default" ng-model="displayRate" btn-radio="'KB/s'">KB/s</label>
 						</div>
 					</div>
+					<div class="controls">
+						<div class="btn-group">
+							<label class="btn btn-default" checked ng-model="displayStyleSheet" btn-radio="'bwmon'">LIGHT</label>
+							<label class="btn btn-default" ng-model="displayStyleSheet" btn-radio="'bwmondark'">DARK</label>
+						</div>

```

Comment:
I just checked with the raw source and there's missing an end `</div>` tag here so I would fix that before merging.

Otherwise the code looks good. I haven't actually tested the layouts myself and check what it looks like on mobile (it tends to be a problem mobile often doesn't have enough space to fit all the elements since it's portrait. That said I'm generally good with experimental features merged into the branch I'll just with hold an actual release package until I've done the testing to verify all the corner cases.